notebooks/700_metrics/701a_aupimo.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>I think it would be good to have a reference section at the end to provide the bibtex file, to encourage citing this work if it is used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>done</p>